lib/appium_lib/device/device.rb

```diff
@@ -15,6 +15,7 @@ module Device
         device_locked?:       'session/:session_id/appium/device/is_locked'
       },
       get:  {
+        get_device_time:        'session/:sessionId/appium/device/system_time',
```

Comment:
@JaniJegoroff https://github.com/appium/node-mobile-json-wire-protocol/blob/master/lib/routes.js#L294 It should be "sessionId" not "session_id".